src/controllers/articleController.js

```diff
@@ -73,8 +73,15 @@ class articleContoller {
   static async getArticle(req, res) {
```

Comment:
This method is for fetching article, any other actions should be abstracted and the method be invoked at the right place.